Gemfile

```diff
@@ -80,7 +80,7 @@ gem "rolify", "~> 5.2" # Very simple Roles library
 gem "rouge", "~> 3.12" # A pure-ruby code highlighter
 gem "rubyzip", "~> 1.2", ">= 1.3.0" # Rubyzip is a ruby library for reading and writing zip files
 gem "s3_direct_upload", "~> 0.1" # Direct Upload to Amazon S3
-gem "sail", "~> 1.5" # Sail is a lightweight Rails engine that brings an admin panel for managing configuration settings on a live Rails app
+gem "sail", "~> 3.2" # Sail is a lightweight Rails engine that brings an admin panel for managing configuration settings on a live Rails app
```

Comment:
What config settings do we use this to manage?

Reply:
Sail isn't fully in place yet so there's no way to direclty configure it, ie we haven't [mount the route](https://github.com/vinistock/sail#installation).